FWCore/Concurrency/interface/WaitingTaskWithArenaHolder.h

```diff
@@ -72,5 +71,29 @@ namespace edm {
     WaitingTask* m_task;
     std::shared_ptr<tbb::task_arena> m_arena;
   };
+
+  template <typename F>
+  auto make_lambda_with_holder(WaitingTaskWithArenaHolder h, F&& f) {
+    return [holder = std::move(h), func = std::forward<F>(f)]() mutable {
+      try {
+        func(holder);
+      } catch (...) {
+        holder.doneWaiting(std::current_exception());
+      }
```

Comment:
Static analyzer complains about `catch (...)`, but because the exception is forwarded, the catch-all is acceptable. Do we have any mechanism to silence static analyzer for accepted cases of catch-all?

Reply:
Not yet. I hope we can extend the use of `CMS_SA_ALLOW`